odoo/addons/base/models/res_partner.py

```diff
@@ -748,7 +719,7 @@ def _commercial_sync_to_descendants(self, fields_to_sync=None):
         if fields_to_sync is None:
             fields_to_sync = self._commercial_fields()
         sync_vals = commercial_partner._convert_fields_to_values(fields_to_sync)
-        sync_children = self.child_ids.filtered(lambda c: not c.is_company)
+        sync_children = self.child_ids
```

Comment:
When discussing specs, this should use VAT... why not finally ? Otherwise globally we always sync everything, which is probably going to lead to information loss.